### PR TITLE
wm3/halla_Sulaiman-JavaScript-core1-week1

### PR DESCRIPTION
**Pull request by Halla-S**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Halla Sulaiman
- Your City: Wolverhampton
- Your Slack Name: Halla Sulaiman

# Homework Details

- Module: JavaScript-core1
- Week: week 1

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: extra/3-magic-8-ball.js [hunk 1]
```diff
@@ -45,8 +45,34 @@
 
 // This should log "The ball has shaken!"
 // and return the answer.
+
 function shakeBall() {
-  //Write your code in here
+  expectedAnswers = [
```
Comment (Professional-Kate) [inline extra/3-magic-8-ball.js:50]: This variable is missing a const or let declaration

**Review by Professional-Kate — COMMENTED** [review]
These are all super simple and petty things, but that's because your code is already amazing. Amazing work, but a couple of small changes 🎉

File: exercises/B-hello-world/exercise.js [hunk 2]
```diff
@@ -1 +1,6 @@
 console.log("Hello world");
+console.log("Hello World. I just started learning JavaScript!.");
+console.log(Hello World. I just started learning JavaScript!.); // got a SyntaxError for this one .
```
[the remark this answers is not in the document]
Comment (Halla-S) [reply exercises/B-hello-world/exercise.js:3]: Yes, Strings should be inside double or single quotes . as you can see the line above is the correct one , this line should've been commented out or deleted as it was originally there as a  apart of the homework  to be corrected I think . I forgot to do so .

File: exercises/F-strings-methods/exercise2.js [hunk 3]
```diff
@@ -1,3 +1,6 @@
-const name = " Daniel  ";
-
-console.log(message);
+const myName = " Daniel  ";
+let myNewName = myName.trim();
+let message1 = myNewName.length;
```
[the remark this answers is not in the document]
Comment (Halla-S) [reply exercises/F-strings-methods/exercise2.js:3]: You are right , thank you .

File: exercises/F-strings-methods/exercise.js [hunk 4]
```diff
@@ -1,3 +1,16 @@
 // Start by creating a variable `message`
+let myName = " Halla ";
+let message = myName.length;
```
[the remark this answers is not in the document]
Comment (Halla-S) [reply exercises/F-strings-methods/exercise.js:3]: Thank you

File: exercises/F-strings-methods/exercise.js [hunk 5]
```diff
@@ -1,3 +1,16 @@
...
+let aboutName = `My name is Halla and my name is ${message} characters long`;
+let aboutName1 =
+  "My name is Halla and my name is " + message + " characters long";
+let myNewName = myName.trim();
+let message1 = myNewName.length;
```
[the remark this answers is not in the document]
Comment (Halla-S) [reply exercises/F-strings-methods/exercise.js:8]: Thank you.

File: extra/3-magic-8-ball.js (hunk 1 above)
Comment (Halla-S) [reply extra/3-magic-8-ball.js:50]: Thank you Kate .As you can see , I declared it first outside of the function which makes it global . Then I changed my mind  and moved it to the inside of the function without declaration , just trying to make it global from the inside of the function to be able to use it in the other function . tried both ways and both work .

**Comment by Krrish-CYF** [comment]
Great work halla.

**Pull request closed without merging**